Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -87,6 +87,10 @@ def genGridOnPoints(data1,gm,deep=True,grid=None,geo=None):
     if grid is None:
       x = g.getLongitude()[:]
       y = g.getLatitude()[:]
+      xm = x[0]
+      xM = x[-1]
+      ym = y[0]
```

Comment:
@doutriaux1 the point I was making is that the developer who is writing the code or modifying the existing code, should do it in the first place. I am hoping that all of us will agree to it: @jbeezley  @chaosphere2112  @williams13